.travis.yml

```diff
@@ -4,12 +4,7 @@ language: ruby
 
 # Ruby build matrix
 rvm:
-  - 2.0
```

Comment:
Maybe we should leave the rubies in here, but just tie jruby to 1.7.x as you have done.